packages/vx-demo/src/docs-v2/examples/vx-streamgraph/useForceUpdate.ts

```diff
@@ -0,0 +1,6 @@
+import { useState } from 'react';
```

Comment:
Should `react` version requirement be `^16.8` to support hooks?

Reply:
ah yes, good catch. there's another dep issue I'm not sure how to handle quiet yet where after all of the examples are refactored, all of the deps should be in the independent sandbox `package.json`s but that doesn't work with `yarn` workspaces because technically they are nested:

```
vx-root/
package.json
...
  packages/
    vx-axis/
      package.json
      ...
    ...
    vx-demo/
      package.json     <-- all deps currently in workspace
      sandboxes/
        sandbox/
          package.json <-- slowly getting moved here, nested inside a workspace
```

will prob figure that out when we get there.